src/GovUk.Education.ExploreEducationStatistics.Data.Processor/local.settings.json

```diff
@@ -4,7 +4,10 @@
     "AzureWebJobsStorage": "UseDevelopmentStorage=true;DevelopmentStorageProxyUri=http://data-storage",
     "CoreStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1;QueueEndpoint=http://data-storage:10001/devstoreaccount1;TableEndpoint=http://data-storage:10002/devstoreaccount1",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "RowsPerBatch": 3000
+    "RowsPerBatch": 3000,
+    "AzureFunctionsJobHost__logging__logLevel__Default": "None",
+    "AzureFunctionsJobHost__logging__logLevel__Function": "Information",
+    "AzureFunctionsJobHost__logging__logLevel__Microsoft": "Warning"
```

Comment:
I assume you have to use this double underscore notation as it's the only way to override the default log configuration from `host.json` in the `local.settings.json`?

This PR says it's to stop the logging locally but isn't it also a problem in other environments where we won't be filtering these messages going into App Insights?

If there's a reason we want this for the local environment fair enough but if not then the equivalent of what we already have in `host.json` i.e.

```
"logging": {
  "logLevel": {
    "default": "Information",
    "Microsoft": "Warning"
  }
}
```

would just become

```
"logging": {
  "logLevel": {
    "default": "None",
    "Function": "Information",
    "Microsoft": "Warning"
  }
}
```

rather than these changes.

Reply:
Talked to Ben about this - switching the `default` from `None` to `Error`. The log spam still isn't on.

Reply:
I checked on AppInsights on Dev, and the logs that were happening locally are also happening there. So I've moved the settings to the host.json so AppInsights also isn't spammed.